src/main/twirl/gitbucket/core/repo/files.scala.html

```diff
@@ -11,7 +11,16 @@
   error: Option[Any] = None)(implicit context: gitbucket.core.controller.Context)
 @import context._
 @import gitbucket.core.view.helpers._
-@html.main(s"${repository.owner}/${repository.name}", Some(repository)) {
+@html.main(
+  if(pathList.isEmpty){
+    if(branch == "master"){
```

Comment:
Use `repository.repository.defaultBranch` instead of the string literal `"master"`